util/slicesx/slicesx.go

```diff
@@ -148,3 +148,23 @@ func FirstEqual[T comparable](s []T, v T) bool {
 func LastEqual[T comparable](s []T, v T) bool {
 	return len(s) > 0 && s[len(s)-1] == v
 }
+
+// MapValues returns the values of the map m.
+//
+// The values will be in an indeterminate order.
+//
+// It's equivalent to golang.org/x/exp/maps.Values, which
+// unfortunately has the package name "maps", shadowing
+// the std "maps" package. This version exists for clarity
+// when reading call sites.
```

Comment:
I feel like this is a weak argument -- it's easy enough to import it as `xmaps`. The allocation optimization seems more compelling, so maybe that should be mentioned first?

Reply:
It's actually not that easy to allocate it as xmaps... goimports doesn't help (as much / as quickly) in that case. And you're back to doing it manually like a caveperson. That's often what bugs me more than the alloc.